blog/forms.py

```diff
@@ -0,0 +1,8 @@
+from django import forms
+from .models import Commentary
+
+
+class CommentForm(forms.ModelForm):
+    class Meta:
+        model = Commentary
+        fields = "__all__"
```

Comment:
don't recommend you use `fields = "__all__"`, it can have bad effect in future while improving code